scripts/functions.sh

```diff
@@ -33,15 +33,11 @@ export INFLUX_PORT="${INFLUX_PORT:-8086}"
 export BUILD_TAG="${BUILD_TAG:-testing}"
 
 # RPC-OpenStack product release, this variable is used in the config playbooks.
-export RPC_PRODUCT_RELEASE="${RPC_PRODUCT_RELEASE:-queens}"
+export RPC_PRODUCT_RELEASE="${RPC_PRODUCT_RELEASE:-master}"
 
 # OSA release branch
 if [ -z ${OSA_RELEASE_BRANCH+x} ]; then
-  if [[ "${RPC_PRODUCT_RELEASE}" != "master" ]]; then
```

Comment:
This conditional should stay so that it sets the OSA branch properly when this master branch is cut o a stable branch some time in the future.